Google.Api.Gax.Rest.IntegrationTests/ClientBuilderBaseTest.cs

```diff
@@ -55,6 +56,23 @@ public async Task ApiKey_NoOtherCredentials()
             await ValidateResultAsync(builder, validator);
         }
 
+        [Fact]
+        public async Task ApiKey_NoOtherCredentials_QuotaProject()
+        {
+            var builder = new SampleClientBuilder 
+            { 
+                ApiKey = SampleApiKey,
+                QuotaProject = SampleQuotaProject 
+            };
+
+            Action<BaseClientService.Initializer> validator = initializer =>
+            {
+                Assert.Equal(SampleApiKey, initializer.ApiKey);
+
+            };
+            await ValidateResultAsync(builder, validator);
```

Comment:
```suggestion
            Action<BaseClientService.Initializer> validator = initializer =>
                Assert.Equal(SampleApiKey, initializer.ApiKey);
            await ValidateResultAsync(builder, validator);
```

Reply:
But should this also be testing that the quota project is used?

Reply:
Yep, added code to check that quota project is used.